static/js/components/email/EmailCompositionType.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import React from 'react';
+import { RadioButton, RadioButtonGroup } from 'material-ui/RadioButton';
+
+import {
+  ONE_TIME_EMAIL,
+  EMAIL_CAMPAIGN
+} from './constants';
+
+export default class EmailCompositionType extends React.Component {
+  props: {
+    emailCompositionType:    ?string,
+    setEmailCompositionType: (b: string) => void,
+  };
+
+  handleRadioClick = (value: string): void => {
+    const { setEmailCompositionType } = this.props;
+    setEmailCompositionType(value);
+  }
+
+  renderEmailCampaign = (): React$Element<*> => (
+    <div className="email-campaign-content">
+      This email will be sent now and in the future whenever users meet the criteria.
+    </div>
+  );
+
+  render() {
+    const { emailCompositionType } = this.props;
+    let radioIconStyle = {'marginRight': '8px'};
```

Comment:
is it tricky to style the radio buttons with sass? I'd like to stay away from inline styles wherever we can.